api.go

```diff
@@ -106,6 +117,21 @@ func (a *API) NumSlots() (int, error) {
 	return n, unmarshalPyON(s, &n)
 }
 
+// Uptime returns FAH uptime
```

Comment:
Sorry to be picky; by Go convention, function comments should end with a period.